src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -1599,4 +1619,9 @@ public class TypeWithPropertiesNestingItsGenericTypeArgument<T>
         public Tuple<int, int?, Tuple<T?>>? Deep4 { get; set; }
         public Tuple<int, int, Tuple<T, int>?>? Deep5 { get; set; }
     }
+
+    public class GenericTypeWithNullableCtor<T>
+    {
+        public GenericTypeWithNullableCtor(T value) { }
+    }
```

Comment:
Could you also include a second type:
```C#
public class GenericNonNullableTypeWithCtor<T> with T : notnull
{
    public GenericNonNullableTypeWithCtor(T value) { }
}
```
and a test confirming that the parameter of `GenericNonNullableTypeWithCtor<object>` is `NotNull`?